rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -0,0 +1,116 @@
+# This package contains macros that provide functionality relating to
+# Software Collections. These macros are not used in default
+# Fedora builds, and should not be blindly copied or enabled.
+# Specifically, the "scl" macro must not be defined in official Fedora
+# builds. For more information, see:
+# http://docs.fedoraproject.org/en-US/Fedora_Contributor_Documentation
+# /1/html/Software_Collections_Guide/index.html
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_remote_execution
+
+%define rubyabi 1.9.1
+
+Summary:    Plugin that brings remote execution capabilities to Foreman
+Name:       %{?scl_prefix}rubygem-%{gem_name}
+Version:    0.0.1
+Release:    1%{?foremandist}%{?dist}
+Group:      Applications/System
+License:    GPLv3
+URL:        http://github.com/theforeman/foreman_remote_execution
+Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires:   foreman >= 1.9.0
+
+%if 0%{?fedora} > 18
+Requires: %{?scl_prefix}ruby(release)
+%else
+Requires: %{?scl_prefix}ruby(abi) >= %{rubyabi}
+%endif
+Requires: %{?scl_prefix}rubygems
+Requires: %{?scl_prefix}rubygem-foreman-tasks >= 0.7.3
+Requires: %{?scl_prefix}rubygem-foreman-tasks < 0.8.0
+Requires: %{?scl_prefix}rubygem-rails >= 3.2.8
+Requires: %{?scl_prefix}rubygem-rails < 3.3.0
+
+%if 0%{?fedora} > 18
+BuildRequires: %{?scl_prefix}ruby(release)
+%else
+BuildRequires: %{?scl_prefix}ruby(abi) >= %{rubyabi}
+%endif
+BuildRequires: foreman-plugin >= 1.9.0
+BuildRequires: foreman-assets
+BuildRequires: %{?scl_prefix}rubygems-devel
+BuildRequires: %{?scl_prefix}rubygems
+BuildRequires: %{?scl_prefix}rubygem-foreman-tasks >= 0.7.3
+BuildRequires: %{?scl_prefix}rubygem-foreman-tasks < 0.8.0
+BuildRequires: %{?scl_prefix}rubygem-rails >= 3.2.8
```

Comment:
Yeah, I'd prefer it but it's not as important.  The reason it fails is because the rubygem-rails RPM has an epoch set, so it's important to compare the version of the virtual rubygem(rails) rather than the actual package EVR.